src/mango_selector_text.erl

```diff
@@ -256,7 +251,9 @@ field_exists_query(Path) ->
     % appened * isntead).
     Parts = [
         {op_fieldname, {[path_str(Path), ":"], "*"}},
-        {op_fieldname, {[path_str(Path), "."], "*"}}
+        % need to extract out the period because mango_doc:parse_field/1
+        % will not accept "name.", also we don't want to escape the .
```

Comment:
Make sure your comments are properly capitalized and grammatically correct. Also, you'll probably want this to be before both and explain why you're escaping the colon, but not the period.